### ISSUE TITLE
fault when building service

### ISSUE THREAD

**Issue by fuale** [tweet9ra/logux-laravel#1]
after 'laravel new blog' and importing your package

laravel 7.x

`Target [tweet9ra\Logux\CommandsProcessorBase] is not instantiable while building [tweet9ra\Logux\App].`

this prevents me from completing the third step from the documentation

**Comment by fuale** [comment]
how fix: create config/logux.php with follow content
```php
<?php

return [
    'control_url' => 'http://localhost:31337',
    'password' => 'abcdefsecreet',
    'protocol_version' => 1
];
```

where problem:
```php
$url = config('logux.control_url');
if (!$url && !$this->app->runningUnitTests()) {
    return;
}
```

i think you must improve docs

**Comment by tweet9ra** [comment]
Fixed in 3.0.2, now you can complete instalation on step 3 without creating config/logux.php manually.
@FaridAkhmedov thank you for reporting :)
